main.js

```diff
@@ -15,7 +15,6 @@ function addGarment(event) {
   toggleGarments('hat', 0);
   toggleGarments('clothes', 1);
   toggleGarments('accessories', 2);
-  toggleGarments('backgrounds', 3);
```

Comment:
This line is no longer necessary, right?

Reply:
Good point. We'll delete it in our next clean-up.